node-require/calculate.js

```diff
@@ -0,0 +1,24 @@
+const a = require('./add');
+const s = require('./subtract');
+const m = require('./multiply');
+const d = require('./divide');
+
+const first = parseInt(process.argv[2]);
+const second = parseInt(process.argv[4]);
```

Comment:
The calculator should still produce the correct output if the inputs contain decimal places.